src/reference/asciidoc/aggregator.adoc

```diff
@@ -279,6 +279,47 @@ A `LockRegistry` is used to obtain a lock for the resolved correlation ID.
 A `DefaultLockRegistry` is used by default (in-memory).
 For synchronizing updates across servers where a shared `MessageGroupStore` is being used, you must configure a shared lock registry.
 
+[[aggregator-deadlocks]]
+===== Avoiding Deadlocks
+
+As discussed above, when message groups are mutated (messages added or released) a lock is held.
+
+Consider the following flow:
+
+====
+[source]
+----
+...->aggregator1-> ... ->aggregatoor2-> ...
```

Comment:
typo

Reply:
Found it. Fixed in my local copy before merge. 

Thank you for review anyway!